docs/en_US/reference/experiment_config.rst

```diff
@@ -501,73 +517,111 @@ type: ``str``
 This can be found in your OpenPAI user settings page.
 
 
-trial cpu number
-----------------
+docker image
+------------
 
-Number of CPUs used by each trial.
+Name and tag of docker image to run the trials.
 
-type: ``int``
+type: ``str``
 
-default: ``1``
+default: ``"msranni/nni:latest"``
 
 
-trial memory size
------------------
+nni manager storage mount point
+-------------------------------
 
-Memory used by each trial.
+`Mount point <path>`_ of storage service (typically NFS) on current machine.
 
 type: ``str``
 
-examples: ``"1gb"``, ``"512mb"``
-
 
-docker image
-------------
+container storage mount point
+-----------------------------
 
-Name and tag of docker image to run the trials.
+Mount point of storage service (typically NFS) in docker container.
 
 type: ``str``
 
-default: ``"msranni/nni:latest"``
+This must be an absolute path.
 
 
 reuse mode
 ----------
 
-Enable reuse mode.
+Enable reuse mode. [TODO: reuse doc]
 
 type: ``bool``
 
 default: ``False``
 
 
-nni manager storage mount point
--------------------------------
+open pai config
+---------------
 
-`Mount point <path>`_ of storage service (typically NFS) on current machine.
+Embedded OpenPAI config file.
 
-type: ``str``
+type: ``Optional[JSON]``
 
 
-container storage mount point
------------------------------
+open pai config file
+--------------------
 
-Mount point of storage service (typically NFS) in docker container.
+`Path`_ to OpenPAI config file.
+
+type: ``Optional[str]``
+
+An example can be found `here <https://github.com/microsoft/pai/blob/master/docs/manual/cluster-user/examples/hello-world-job.yaml>`__
+
+
+AmlConfig
+=========
+
+Detailed `here <../TrainingService/AMLMode.rst>`__.
+
+
+platform
+--------
+
+Constant string ``"aml"``.
+
+
+docker image
+------------
+
+Name and tag of docker image to run the trials.
 
 type: ``str``
 
-This must be an absolute path.
+default: ``"msranni/nni:latest"``
```

Comment:
Is image name optional key since it has default value?

Reply:
Yes, having default value means "optional", in human's language.
And "Optional" in Python language means "or None".